ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -221,6 +222,33 @@ export const displayTierBadgeByRequiredModule = (
   );
 };
 
+export const getEraNote = (profile: Profile): string => {
+  // prettier-ignore
+  const tierDisplayNames = Object.keys(AccessTierDisplayNames).map(function (key) {
```

Comment:
our standard is arrow-style which we may start enforcing soon via the linter, like:
```
.map(key => AccessTierDisplayNames[key])
```

Reply:
Done.